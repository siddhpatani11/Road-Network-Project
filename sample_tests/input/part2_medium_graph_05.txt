35 189
0 2 17 B
0 11 25 B
0 17 7 G
0 20 26 B
0 27 47 R
0 29 22 B
0 31 40 G
1 8 48 G
1 11 7 G
1 14 5 R
1 18 22 G
1 25 6 G
1 29 15 B
2 1 15 G
2 11 1 R
2 12 21 R
2 15 7 B
2 18 22 R
2 22 25 G
2 26 34 B
2 28 34 R
2 32 34 R
3 6 46 R
3 9 40 G
3 17 32 G
3 27 9 G
4 1 21 G
4 5 31 B
4 7 16 G
4 16 13 R
4 17 18 B
4 26 8 G
4 30 24 R
5 3 18 G
5 12 12 G
5 13 50 B
5 16 18 R
5 34 14 B
6 1 31 R
6 2 3 B
6 3 29 R
6 7 15 B
6 9 48 R
6 14 28 B
6 15 44 R
6 32 46 B
6 34 42 R
7 28 4 R
7 32 1 R
8 16 14 B
8 17 11 R
8 20 41 G
8 25 21 B
8 31 22 R
8 34 33 G
9 10 25 R
9 12 24 G
9 22 7 G
9 27 49 G
10 22 2 R
10 27 4 B
10 31 45 R
11 6 9 R
11 7 15 R
11 8 3 G
11 23 25 B
11 27 37 G
12 9 30 B
12 19 35 R
12 21 3 G
12 24 27 B
12 25 15 G
12 28 51 G
12 33 24 B
13 2 17 G
13 6 14 G
13 12 30 G
13 17 14 R
13 27 24 B
14 18 2 G
14 21 10 G
14 25 48 R
14 30 22 R
15 2 47 B
15 8 44 R
15 9 21 G
15 12 5 G
15 28 50 B
16 12 24 B
16 15 15 G
16 26 19 B
16 29 18 R
16 34 36 R
17 3 40 B
17 7 10 B
17 8 39 R
17 10 22 B
17 23 15 G
17 32 44 G
18 11 26 R
18 13 41 B
18 15 7 G
18 22 35 G
18 24 45 B
18 26 26 B
19 2 21 B
19 8 18 B
19 10 47 G
19 17 45 R
19 20 16 B
19 21 34 R
20 1 26 B
20 2 30 R
20 5 37 R
20 33 19 G
20 34 24 B
21 5 7 G
21 11 46 B
21 12 18 B
21 13 7 B
22 9 40 B
22 14 39 B
22 15 31 B
22 19 51 B
22 26 14 R
22 30 11 R
23 8 2 G
23 14 50 G
23 25 11 B
24 3 6 G
24 8 27 R
24 12 11 B
24 22 29 R
24 25 47 R
24 31 40 G
24 33 15 R
25 12 24 B
25 19 6 R
25 21 48 R
25 27 7 G
26 3 32 R
26 5 23 R
26 6 39 R
26 7 11 B
26 10 14 B
26 16 2 G
26 19 21 G
26 28 32 R
27 11 40 R
27 17 20 G
27 28 36 B
27 30 47 G
27 33 6 G
28 12 26 B
28 32 24 G
29 2 20 R
29 4 24 G
29 11 47 B
29 13 31 B
29 17 50 R
29 23 24 G
30 2 10 G
30 4 5 G
30 9 39 R
30 15 43 G
30 20 9 R
30 23 31 R
30 29 12 G
31 0 23 G
31 5 13 B
31 7 25 B
31 9 30 R
31 28 39 G
31 29 37 G
31 34 2 B
32 15 7 B
32 28 8 R
32 31 26 G
33 3 42 R
33 9 40 R
33 14 9 B
33 15 42 R
33 20 44 G
33 22 49 G
33 29 2 G
34 16 35 B
34 21 11 G
34 25 17 B
34 28 34 B
