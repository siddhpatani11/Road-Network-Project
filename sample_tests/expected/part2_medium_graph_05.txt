6 R 151 R 31 B 3 R 29 B 55 B 55 - 0 B 15 B 7 R 48 B 81 B 4 B 24 B 34 B 28 B 32 B 21 B 48 B 25 B 34 R 135 B 27 R 55 R 63 R 76 B 28 R 79 R 38 B 19 B 39 B 50 B 42 B 16 R 73 R 42 
